docs/documentation/server_admin/topics/organizations/authenticating-members.adoc

```diff
@@ -0,0 +1,99 @@
+
+* `browser`
+
+* `first broker login`
+
+The main change to the `browser` flow is that it defaults to an identity-first login so that users are identified before prompting for their credentials.
+In regard to the first broker login flow, the main change there is to automatically add the user as an organization member once they authenticate through the identity provider associated with an organization and successfully complete flow.
+
+The decision to whether or not an identity-first login should happen is based on the availability of any organization in a realm.
+If no organizations exist yet, the user will follow the usual steps to authenticate using both username and password,
+or any other step configured in the browser flow. Otherwise, the user will be asked first for a username or email to continue authenticating to a realm.
+
+The identity-first login main goal is to identify the user:
+
+* Is the user an existing or a new user?
+* Is the user a member of any organization within a realm?
+* If an organization member, is the user linked to any identity provider associated with the organization?
+
+Depending on the outcome when identifying the user, the authentication flow will change to either proceed with the authentication
+by asking for the user's credentials or eventually redirect the user automatically to authenticate within the organization security
+boundaries through an identity provider.
+
+== Understanding the identity-first login
+
+In addition to identifying the user once the username is provided, the identity-first login is also responsible to:
+
+* Match an email domain to an organization
+* Decide if the authentication flow should continue or not if an account already exists for the username provided
+* Decide how the user should be authenticated depending on how the domains and the identity providers are configured to an organization
+* Seamless authenticate users through an identity provider associated with an organization if the email domain matches the domain set to the identity provider
+
+The identity-first login provides the very same capabilities when using the usual login page with both username and
+password fields. The users can still self-register by clicking on the register link or choose from any of the identity or social
+brokers available at the realm not linked to any organization.
+
+If a user that does not exist tries to authenticate using an email domain that matches an organization domain,
+the identity-first login page will be shown again and indicate that the username provided is not valid.
+At this point, there is no reason to ask the user for credentials in a second step.
+
+There are several ways to register the user so that he can authenticate to the realm and eventually join an organization.
+
+If the realm has the self-registration setting enabled, the user can click on the `Register` link at the identity-first login page
```

Comment:
```suggestion
If the realm has the self-registration setting enabled, the user can click the *Register* link at the identity-first login page
```

Reply:
We use "click" not "click on"
Also, we use *bold* for items in the user interface